src/content/docs/releases/index.mdx

```diff
@@ -5,18 +5,16 @@ description: Learn about the compatibility, code changes, and public API for Ado
 
 import Aside from '@components/Aside.astro';
 
-Adobe organizes storefront application releases into logical suite versions and follows a regular release schedule (approximately once a month). However, Adobe may release updates more frequently to deliver new features or address critical issues. 
+Adobe organizes storefront application releases into logical suite versions and follows a regular release schedule (approximately every other month). Adobe may release updates more frequently to deliver new features or address critical issues. 
 
 ## Compatibility table
 
-The following table shows the compatibility between all of the storefront release items by suite version:
+The following table shows the compatibility between the storefront release items by suite version:
 
-| Storefront suite | | 1.0.0 |
-|---|---|---|
-| **Adobe Commerce** | | 2.4.7 |
-| **Storefront Compatibility Package** | | 4.7.0 |
-| **Drop-in SDK** | | `@dropins/tools@0.38.0`<br/>`@dropins/build-tools@0.1.1` |
-| **Drop-in components** | | `@dropins/storefront-cart@1.0.1`<br/>`@dropins/storefront-checkout@1.0.0`<br/>`@dropins/storefront-order@1.0.0`<br/>`@dropins/storefront-payment-services@1.0.1`<br/>`@dropins/storefront-pdp@1.0.0`<br/>`@dropins/storefront-user-account@1.0.0`<br/>`@dropins/storefront-user-auth@1.0.0` |
+| Storefront suite | | **Adobe Commerce** | **Storefront Compatibility Package** | **Drop-in SDK** | **Drop-in components** |
+|---|---|:---:|:---:|---|---|
+| December 2024 | | `2.4.7` | `4.7.0` |  <ul><li>`@dropins/tools@0.38.0`</li><li>`@dropins/build-tools@0.1.1`</li></ul> | <ul><li>`@dropins/storefront-cart@1.0.1`</li><li>`@dropins/storefront-checkout@1.0.0`</li><li>`@dropins/storefront-order@1.0.0`</li><li>`@dropins/storefront-payment-services@1.0.1`</li><li>`@dropins/storefront-pdp@1.0.0`</li><li>`@dropins/storefront-user-account@1.0.0`</li><li>`@dropins/storefront-user-auth@1.0.0`</li></ul> |
+| April 2025 | | `2.4.7` | `4.7.1` |  <ul><li>`@dropins/tools@1.0.0`</li><li>`@dropins/build-tools@0.2.0`</li></ul> | <ul><li>`@dropins/storefront-cart@1.2.2`</li><li>`@dropins/storefront-checkout@1.3.0`</li><li>`@dropins/storefront-order@1.0.5`</li><li>`@dropins/storefront-payment-services@1.0.1`</li><li>`@dropins/storefront-pdp@1.0.0`</li><li>`@dropins/storefront-user-account@1.0.4`</li><li>`@dropins/storefront-user-auth@1.0.6`</li></ul> |
```

Comment:
@supernova-at   Tools is set to ` "@dropins/tools": "^0.42.0"`

Reply:
good catch, thank you!

Reply:
Fixed in c1cd3a42cdc2ab2d66015c01fe5ba217348bd1ae.